packages/lib/services/plugins/utils/manifestFromObject.ts

```diff
@@ -72,6 +74,7 @@ export default function manifestFromObject(o: any): PluginManifest {
 	};
 
 	validatePluginId(manifest.id);
+	validatePluginPlatforms(manifest.platforms);
```

Comment:
```suggestion

```

Either update `validatePluginPlatforms` to allow unknown platform names, or don't use it here (i.e. only use it in the plugin repo CLI). Currently this line will prevent plugin authors from specifying currently unknown platforms while continuing to support older versions of Joplin. This would be an issue if we eventually add a web client with plugins or add plugins to the CLI app.

Reply:
Updated to allow unknown platform names.